index.html

```diff
@@ -220,6 +220,32 @@ <h3 class="small__resume_title">Skills</h3>
         </section>
 
         <!-- Contact Form -->
+        <section class="contacts">
+            <h2 class="contact_heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyryqo" class="contact_form" method="post">
+                <ul class="form_items">
+                    <li class="form_list">
+                        <input type="text" class="name_input" placeholder="Fullname" maxlength="30">
+                    </li>
+                    <li class="form_list">
+                        <input type="email" name="user_email" class="email_input" placeholder="Email">
+                    </li>
+                    <li class="form_list">
+                        <textarea cols="30" rows="10" class="message_input" placeholder="Your Message"></textarea>
+                    </li>
+                    <li class="form_buttons">
+                        <button type="submit" class="form_submit">Get in Touch</button>
+                    </li>
+                </ul>
+
+                <ul class="social__media">
+                    <li class="handles"><a href="http://"><img src="./images/social media/github.svg" alt="github" srcset=""></a></li>
+                    <li class="handles"><a href="http://"><img src="./images/social media/Linkedin icon.svg" alt="Linkedin" srcset=""></a></li>
+                    <li class="handles"><a href="http://"><img src="./images/social media/twitter.svg" alt="Twitter" srcset=""></a></li>
+                    <li class="handles"><a href="http://"><img src="./images/social media/medium.svg" alt="twitter" srcset=""></a></li>
+                </ul>
+            </form>
+        </section>
```

Comment:
- [ OPTIONAL] Please add the `linebar` at the bottom of your page to reflect the Figma design as the prior reviewer overlooked it. :wink: 
![LINE](https://user-images.githubusercontent.com/88894525/194317000-a90ed68f-4302-41dc-8fe9-2d6a338b4c62.png)